src/js/components/Box/box.stories.js

```diff
@@ -130,7 +130,11 @@ const BorderBox = () => (
           'inset',
           'outset',
         ].map(type => (
-          <Box key={type} pad="small" border={{ type, size: 'medium' }}>
+          <Box
+            key={type}
+            pad="small"
+            border={{ type, size: 'medium', style: type }}
```

Comment:
Two questions:

I don't see either `type` or `style` in the the documentation for Box https://v2.grommet.io/box#border, should we have it there?

Why do we need both `type` and `style` keys?

Reply:
oups the type is just leftover i forgot to remove. on the other hand i don't know why the border style is not appearing on the grommet-site. it is present on the doc.js file.

Reply:
https://github.com/grommet/grommet-site/pull/101
updating the grommet site.

Reply:
but I kind of like this PR better for updating grommet-site https://github.com/grommet/grommet-site/pull/100/files

Reply:
Removed the extra type prop